perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -897,10 +924,105 @@ def Cleanup(self, vm):
     vm.RemoteCommand('/usr/bin/yes | sudo pip uninstall python-gflags')
 
 
+class SwiftStorageBenchmark(object):
+  """OpenStack Swift version of storage benchmark."""
+
+  def __init__(self):
+    self.swift_command_prefix = ''
+
+  def Prepare(self, vm):
+    """Prepare vm with swift cli and create a swift container using vm.
+
+    Args:
+      vm: The vm being used to run the benchmark.
+    """
+    # Upgrade to latest urllib3 version fixes SSL SNMI error
+    vm.InstallPackages('python-urllib3')
+    vm.RemoteCommand('sudo pip install python-keystoneclient')
+    vm.RemoteCommand('sudo pip install python-swiftclient')
+    vm.bucket_name = 'pkb%s' % FLAGS.run_uri
+    self.swift_command_prefix = _MakeSwiftCommandPrefix(vm.client.url,
+                                                        vm.client.tenant,
+                                                        vm.client.user,
+                                                        vm.client.password)
+    vm.RemoteCommand('swift %s post %s' % (self.swift_command_prefix,
+                                           vm.bucket_name))
+
+  def Run(self, vm, metadata):
+    """Run upload/download on vm with swift cli.
+
+    Args:
+      vm: The vm being used to run the benchmark.
+      metadata: the metadata to be stored with the results.
+
+    Returns:
+      A list of lists containing results of the tests. Each scenario outputs
+      results to a list of the following format:
+        name of the scenario ,value, unit of the value, metadata
+        e.g.,
+        'one byte object upload latency p50', 0.800, 'seconds', 'storage=swift'
+
+      Then the final return value is the list of the above list that reflect
+      the results of all scenarios ran here.
+    """
+    metadata[SWIFTCLIENT_LIB_VERSION] = _GetClientLibVersion(
+        vm, 'python-swiftclient')
+    results = []
+    scratch_dir = vm.GetScratchDir()
+
+    # CLI tool based tests
+
+    clean_up_container_cmd = ('swift %s delete %s'
+                              % (self.swift_command_prefix, vm.bucket_name))
+    upload_cmd = ('time '
+                  'swift %(prefix)s upload %(container)s %(directory)s'
+                  % {'container': vm.bucket_name,
+                     'directory': '%s/run/data/' % scratch_dir,
+                     'prefix': self.swift_command_prefix})
+
+    clean_up_local_temp_cmd = 'rm %s/run/temp/*' % scratch_dir
```

Comment:
I had a question for the `rm` command on line 983.

Shouldn't the `rm` command include the `-r` flag since there could be other directories under `${scratch_dir}/run/temp/` ?